Example/.vscode/settings.json

```diff
@@ -1,6 +1,7 @@
 {
     "swift.sourcekit-lsp.backgroundIndexing": "on",
     "swift.sourcekit-lsp.trace.server": "messages",
+    "swift.sourcekit-lsp.serverPath":  "/Users/mostafa.essam/Library/Developer/Xcode/DerivedData/sourcekit-lsp-hjlrhelrwpehrpezdvulxehkrvsc/Build/Products/Debug/sourcekit-lsp",
```

Comment:
We should remove this line since this will not work by default on other machines